chapters/ch15/1.llm-exploration.ipynb

```diff
@@ -12,7 +12,18 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Noting the torch output here as well (mentioned in earlier chapter feedback). Not sure if we have version of torch that is too new (since there's no version specified in requirements.txt currently, which we should add). Possible there's nothing we can do about this warning if we can't get the dependencies aligned, but noting for whenever the visiontorch version is selected.

Reply:
Noted. There are many warnings regarding dependency versions similar to this throughout the codebase.